Update two_nodes.md (#6)
fluidos-project/Docs: docs/workflows/same_domains/two_nodes.md

# Two nodes, same domain

> The complete workflow, step by step of the interaction between two nodes belonging to the same domain is the following.

Author: ```suggestion
The complete workflow showing the interaction between two nodes belonging to the same domain is the following.
```
